projects/real_world_ovmm/build_3d_map.py

```diff
@@ -468,6 +473,10 @@ def main(
             voxel_map.show(instances=True)
         voxel_map.get_2d_map(debug=show_maps)
 
+        # NOTE: you can set this to True to just sample random goals entirely
+        # TODO: add as a command line option
+        random_goals = False
+
         if explore:
```

Comment:
nit: I usually like to make boolean dtype clear from naming, e.g. `is_explore`. To prevent accidental errors where explore might be set to zero, and empty string, etc.

Reply:
Done; renamed this to run_explore